docker/common/install_tensorrt.sh

```
@@ -33,6 +43,19 @@ for i in "$@"; do
 done
 
 NVCC_VERSION_OUTPUT=$(nvcc --version)
+
+if [[ $(echo $NVCC_VERSION_OUTPUT | grep -oP "\d+\.\d+" | head -n 1) == ${CUDA_VER_12_9} ]]; then
```

Shellcheck (0.11.0)

[warning] Quote the right-hand side of == in [[ ]] to prevent glob matching.

(SC2053)